__tests__/requests/get-articles.test.ts

```diff
@@ -5,7 +5,7 @@ import { getArticles } from '../../src/api-requests/get-articles';
 
 const expectedQuery = gql`
     query Articles {
-        articles @rest(type: "Article", path: "/articles") {
+        articles @rest(type: "Article", path: "/articles?_limit=-1") {
```

Comment:
Good catch on these!

Reply:
Will we ever have a situation where we get too many articles for the site to handle? If this is the case, we should cap it anyway at say, 10k? Though this would probably be unlikely.

Reply:
highly unlikely from strapi.